.github/workflows/cpp_tests.yml

```diff
@@ -27,7 +27,7 @@ jobs:
         cc: gcc
         cxx: g++
         build-type: Debug
-        run-test: true
+        run-test: false
```

Comment:
I would say running these tests by default would be the way to go for all PRs - was there any reason to disable?

Reply:
When running tests via ctest, we can not pass individual gtest flags, e.g. for disabling specific test methods. So we run it via the Makefile after building. This is also the reason it's now a single binary again, so it's easier to execute.

Reply:
Perfect - nice one

Reply:
@COM8 just on this question above, my question was more around currently these tests not running automatically as part of the PRs, is there a reason for these to be disabled? Was your answer related to that?

Reply:
Yes, since I'm using a community CMake extension, they need to be merged first, so they are trusted and will be run.